wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping/en.md

```diff
@@ -1,18 +1,18 @@
 # Code of conduct for modding and mapping
 
-The **Code of Conduct for Modding and Mapping** is a set of rules and guidelines that apply to the entirety of the osu! [modding](/wiki/Modding) and [mapping](/wiki/Beatmapping) ecosystem and an extension to the [osu! community rules](/wiki/Rules) and [general ranking criteria](/wiki/Ranking_Criteria). In order to ensure that mapping and modding discussions in threads take place in a constructive, positive and productive environment, a code of conduct is crucial to get everyone on the same page. When participating in the modding and mapping ecosystem, it is mandatory to follow this Code of Conduct. Misconduct that violates these rules might lead to [penalties issued to your account](/wiki/Silence).
+The **Code of conduct for modding and mapping** is a set of rules and guidelines that apply to the entirety of the osu! [modding](/wiki/Modding) and [mapping](/wiki/Beatmapping) ecosystem and an extension to the [osu! community rules](/wiki/Rules) and [general ranking criteria](/wiki/Ranking_Criteria). In order to ensure that mapping and modding discussions in threads take place in a constructive, positive and productive environment, a code of conduct is crucial to get everyone on the same page. When participating in the modding and mapping ecosystem, it is mandatory to follow this code of conduct. Misconduct that violates these rules might lead to [penalties issued to your account](/wiki/Silence).
 
-## Behaviour and Conduct
+## Behaviour and conduct
 
-- The goal of modding is to help a mapper improve their work so that it can be pushed towards [Ranked](/wiki/Beatmap/Category#ranked) status if the mapper chooses to do so. If you're not interested in helping people, steer clear of the discussion to avoid *hurting them*.
+- The goal of modding is to help a mapper improve their work so that it can be pushed towards [ranked](/wiki/Beatmap/Category#ranked) status if the mapper chooses to do so. If you're not interested in helping people, steer clear of the discussion to avoid *hurting them*.
 - When modding, consider who the post is directed at, what you aim to achieve by posting and what response you expect to receive. This will ensure your best intentions are conveyed and your post is easy to understand.
   - Don't shy away from new mappers and modders - these people are the ones who can benefit the most from your input! Be patient, understanding, and provide as much explanation as possible so that they can learn and improve more quickly
```

Comment:
also em dashes

```suggestion
  - Don't shy away from new mappers and modders—these people are the ones who can benefit the most from your input! Be patient, understanding, and provide as much explanation as possible so that they can learn and improve more quickly
```